tests/e2e/buildspec.yml

```diff
@@ -7,7 +7,7 @@ phases:
   build:
     commands:
       - sed -i -e 's/^CONFIG_BACKEND_URL=.*$/CONFIG_BACKEND_URL=https:\/\/api.rudderlabs.com/' build/docker.env
-      - go run tests/helpers/tomlmerge/toml_merge.go config/config.toml tests/e2e/config_overrides.toml > tests/e2e/config.toml
+      - go run tests/helpers/tomlmerge/toml_merge.go config/config.yaml tests/e2e/config_overrides.toml > tests/e2e/config.toml
```

Comment:
this will not work yaml will not work for tool merge and moreover we will not have yaml in this path

Reply:
removing this line should do no harm right?

Reply:
Copy config_overrides.toml into tests/e2e/config.toml